### DISCUSSION TITLE
Is it Possible to have a Map File for the Entire World?

### DISCUSSION THREAD

**Discussion by SereneK1ng** [mapsforge/mapsforge#1245] [Q&A] [answered]
I was just wondering if there was a way to have a map file for the entire world, or just one for each country. Thank you.

**Comment by devemux86** [comment, marked as answer]
There is a world.map in [Mapsforge server](https://download.mapsforge.org/maps/world/).

You can create a more detailed one with more layers following the [map creation guide](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md).
